packages/web/src/fixtures/dev-kit/hooks.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { renderHook } from '@testing-library/react-hooks'
+import { useGetTotalRewardsAmount } from './hooks'
+import useSWR from 'swr'
+import { toNaturalNumber } from 'src/fixtures/utility'
+
+jest.mock('swr')
+jest.mock('src/fixtures/utility')
```

Comment:
In order to mock "toNaturaruNumber" in line 23, we specify utility in jest.Mock here.